src/SkiaSharp.QrCode/QRCodeRenderer.cs

```diff
@@ -19,8 +19,9 @@ public class QRCodeRenderer : IDisposable
         /// <param name="area">The area.</param>
         /// <param name="data">The data.</param>
         /// <param name="qrColor">The color.</param>
-
-        public void Render(SKCanvas canvas, SKRect area, QRCodeData data, SKColor? qrColor)
+        /// <param name="icon">The icon.</param>
+        /// <param name="iconSizePercent">The icon size.</param>
+        public void Render(SKCanvas canvas, SKRect area, QRCodeData data, SKColor? qrColor, SKBitmap icon = null, int iconSizePercent = 15)
         {
```

Comment:
How about adding `IconData` represent icon info?

```csharp
public class IconData
{
    public SKBitmap Icon { get; set; }
    public int IconSizePercent { get; set; } = 15;
}
```

So signature will be like this?
```csharp
public void Render(SKCanvas canvas, SKRect area, QRCodeData data, SKColor? qrColor, IconData icon)
```

Reply:
Sounds like a great idea, i just implemented that 👍 
Thanks for the suggestion!

Reply:
btw should i move the ECCLevel class into the newly created models folder?

Reply:
No, moving ECCLevel to SkiaSharp.QrCode.Models namespace is breaking change.

Reply:
no, moving ECCLevel class to SkiaSharp.QrCode.Models namespace is breaking change.